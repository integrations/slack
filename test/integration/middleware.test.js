const request = require('supertest');

const { probot } = require('.');

describe('Middleware', () => {
  test('Parse action payload returns 400 for invalid format', async () => {
    await request(probot.server).post('/slack/actions').send({
      team: 'T01234',
      callback_id: '1234',
    })
      .expect(400);
  });
});
