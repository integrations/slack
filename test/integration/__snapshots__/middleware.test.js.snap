// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Middleware Parse action payload returns 400 for invalid format 1`] = `Set {}`;

exports[`Middleware Parse action payload returns 400 for invalid format 2`] = `Map {}`;
